arch/arm/core/aarch64/Kconfig

```diff
@@ -58,6 +58,12 @@ config IPM_CONSOLE_STACK_SIZE
 
 if CPU_CORTEX_A
 
+config ARMV8_A_NS
+	bool "ARMv8-A Normal World (Non-Secure world of Trustzone)"
+	help
+	  This option signifies that Zephyr is running in Trustzone
+	  Normal World.
```

Comment:
Should we make it explicit Zephyr is entered in Trustzone Non-secure state.
Overall the PR looks good.

Reply:
I am fine with default NS or S-EL3, just default y or select it. But it may break others which already use zephyr in EL3. So I would not change it to default currently. Until we could migrate all arm64 boards to NS, we could make it default. @carlocaione Any comments?

Reply:
I just meant to update the config help  "This option signifies that Zephyr is running in Trustzone
	  Normal World."   to  "This option signifies that Zephyr is **entered** in Trustzone
	  Non-Secure state".

Default option is better to leave at EL3 ie "n" and change later depending on how all platforms use it.

Reply:
@sandeepbrcm  Updated. Please help approve this PR.